crates/extricrate/src/lib.rs

```diff
@@ -171,6 +195,21 @@ pub mod dependencies {
         None
     }
 
+    fn mod_to_path(crate_root: &Path, ancestors: &[String], ident: &Ident) -> Option<PathBuf> {
```

Comment:
Maybe add a note that we don't support `path` attributes on mods (and I don't think we should for now, it would complicate the code a lot)